### DISCUSSION TITLE
Pade: File upload by dropping pictures

### DISCUSSION THREAD

**Discussion by zxpert34** [igniterealtime/openfire-pade-plugin#239] [Q&A] [answered]
Received the above message 
```
Sorry, looks like file upload is not supported by your server.
```
when attempting to drop a picture into the chat.

Is this supported with Openfire 4.6.1 and Pade 1.3.0?  If so, what am I missing?

Thanks for any help!

**Comment by deleolajide** [comment, marked as answer]
install openfire httpfileupload plugin https://github.com/igniterealtime/openfire-httpFileUpload-plugin

**Comment by ClaudeStabile** [comment]
FYI,

I confirm i used httpsfileupload plugin with Padé 1.3.0 & 1.3.1 + OF 4.6.0 and 4.6.1
Check logs, ensure permissions to upload file is granted. Give a try with a basic XMPP client as Xabber or pidgin to upload files, if this work drag & drop in Padé will work too

Hope this helps

**Comment by zxpert34** [comment]
Yes, it works fine after loading the plugin!  When I first installed the plugin, it wasn't working, but was great after restarting Openfire.

Thanks for the help!!!
